OpenRA.Mods.Common/Widgets/Logic/CreditsLogic.cs

```diff
@@ -57,7 +60,9 @@ public CreditsLogic(Widget widget, ModData modData, Action onExit)
 			if (hasModCredits)
 			{
 				var modCredits = modData.Manifest.Get<ModCredits>();
-				modLines = ParseLines(modCredits.ModCreditsFile);
+				modLines = ParseLines(
+					modData.DefaultFileSystem.Open(modCredits.ModCreditsFile).ReadAllLines()
```

Comment:
The `ReadAllLines` can be done inside `ParseLines` to reduce duplication.

Reply:
Also,

> Widgets\Logic\CreditsLogic.cs(45,4): error SA1009: Closing parenthesis should not be preceded by a space. [C:\projects\openra\OpenRA.Mods.Common\OpenRA.Mods.Common.csproj]

Reply:
@reaperrr I don't get that error for some reason. Any idea why?